src/kit/Tags.tsx

```diff
@@ -131,7 +130,6 @@ const Tags: React.FC<Props> = ({ compact, disabled = false, ghost, tags, onActio
                 key={tag}
                 ref={editInputRef}
                 size="small"
-                style={{ width: inputWidth }}
```

Comment:
this style was used to set the input width in tag like

<img width="327" alt="image" src="https://github.com/determined-ai/hew/assets/109113616/e272c5fa-25f1-4af9-a125-7948f0641a15">

but after removing `style`, input width gets too long like the following image. @johnkim-det is it expected? 

<img width="353" alt="image" src="https://github.com/determined-ai/hew/assets/109113616/6ffbaaa2-d192-45bb-88bd-d91da7489a14">

but now

Reply:
can we add a prop specifically for setting `width`?

Reply:
done